docs/tutorials/tacho-motors.md

```diff
@@ -92,6 +92,8 @@ it very fast). Press `^C` when you are done. Note: `\033[0G` is an escape code
 that moves the cursor back to the beginning of the line so that we don't fill up
 the screen with numbers.
 
+The range of `position` attribute is between -2,147,483,648 and +2,147,483,647. It is unlikely that you will ever reach the limit.
```

Comment:
Might be good to add the unit of measurement here. It is "tachometer counts" (not degrees).